### ISSUE TITLE
Polacode not using my set font

### ISSUE THREAD

**Issue by yankyhoffman** [octref/polacode#127]
I am using vscode on Ubuntu (19.04), with font `Office Code Pro D` and theme `Monokai`, and Polacode is not using my set font.

here is a screenshot of my issue 
![image](https://user-images.githubusercontent.com/41123800/66154446-74bb0680-e5eb-11e9-8046-1b20454022d8.png)
(Note the double underscores)

_Originally posted by @yankyhoffman in https://github.com/octref/polacode/issues/19#issuecomment-538071051_

**Comment by kufii** [comment]
does it work if you set `editor.fontLigatures: true`?

**Comment by yankyhoffman** [comment]
Nope, I tried setting `fontLigatures` to `true` but it didn't help

**Comment by Maxwell-Lyu** [comment]
WARNING: This is not a solution, it's a hack
If you REALLY want to use your font, open this file
`C:\Users\[YOUR USER NAME HERE]\.vscode\extensions\pnp.polacode-0.3.4\webview\index.html`
> This is the extension directory under windows 10. However, other systems(Ubuntu, etc. ) may have a similar location

Add your font, right after `font-family: ` at line 74, remember to add ", ", and if there are any space in your font name, use quotation marks.
`  <div id="snippet" style="color: #d8dee9;background-color: #2e3440;font-family: 'your font here',SFMono-Regular, ......`
> Line number may differ, but `div id` must be "sinppet"

Restart disable the extension, re-enable it, then restart VSCode, it works.

**Comment by Maxwell-Lyu** [comment]
> Nope, I tried setting `fontLigatures` to `true` but it didn't help

WARNING: This is not a solution, it's a hack, however works
If you REALLY want to use your font, open this file
`C:\Users\[YOUR USER NAME HERE]\.vscode\extensions\pnp.polacode-0.3.4\webview\index.html`
> This is the extension directory under windows 10. However, other systems(Ubuntu, etc. ) may have a similar location

Add your font, right after `font-family: ` at line 74, remember to add ", ", and if there are any space in your font name, use quotation marks.
`  <div id="snippet" style="color: #d8dee9;background-color: #2e3440;font-family: 'your font here',SFMono-Regular, ......`
> Line number may differ, but `div id` must be "sinppet"

Disable the extension, re-enable it, then restart VSCode, it works.

**Comment by yankyhoffman** [comment]
Thanks for the hack! works perfectly!

**Comment by gowrizrh** [comment]
Didn't expect, but this hack works like a charm.

**Comment by kufii** [comment]
my PR fixes this, but while we're waiting for this to be merged, I made a project inspired by polacode that you can use while we wait for it to be merged?

https://marketplace.visualstudio.com/items?itemName=adpyke.codesnap

**Comment by xy137** [comment]
> my PR fixes this, but while we're waiting for this to be merged, I made a project inspired by polacode that you can use while we wait for it to be merged?
> 
> https://marketplace.visualstudio.com/items?itemName=adpyke.codesnap

great alternative for anybody else running into issues

**Comment by techieshark** [comment]
Hack from @Maxwell-Lyu worked. Note you can use "Extensions: Open Extensions Folder" from VS Code command palette then navigate to `pnp.polacode-*` folder and continue editing the file as described by Maxwell above.

**Comment by adi-works** [comment]
Why isn't it merged? really annoying bug tbh. I rather install a fork than mess around with hacks for something so simple. Thanks @kufii.
